.gitignore

```diff
@@ -0,0 +1,7 @@
+.bundle
+Gemfile.lock
```

Comment:
We had a conversation about this in person, I think we should commit it as per the bundler manpage. Especially since different versions of testing libraries (such as rubocop) could introduce a build failure if they're not the same version.